model_compression_toolkit/common/quantization/quantizers/power_of_two_quantizer.py

```diff
@@ -42,7 +42,7 @@ def symmetric_uniform_quantizer(tensor_data: np.ndarray,
     """
     threshold = quantization_params.get(THRESHOLD)
     if threshold is None:
-        raise Exception('\'weights_threshold\' parameter must be defined in \'quantization_params\'')
+        raise Exception("'weights_threshold' parameter must be defined in 'quantization_params'")
```

Comment:
Why this is true weights thresholds?
It should be threshold no?

Reply:
This function is used for weights quantization only